charts/freshrss/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+appVersion: 1.17.0
+description: FreshRSS is a self-hosted RSS feed aggregator
+name: freshrss
+version: 1.0.0
+keywords:
+  - freshrss
+  - rss
+home: https://github.com/k8s-at-home/charts/tree/master/charts/freshrss
+icon: https://github.com/FreshRSS/FreshRSS/blob/master/docs/img/FreshRSS-logo.png?raw=true
+sources:
+  - https://github.com/FreshRSS/FreshRSS
+  - https://hub.docker.com/r/linuxserver/freshrss
+maintainers:
+  - name: npawelek
+    email: nathan@nathanpawelek.com
+dependencies:
+  - name: common
+    repository: https://k8s-at-home.com/charts/
+    version: ^1.6.1
```

Comment:
change this version to `1.6.1` (without the carrot)

Reply:
there's a huge PR I submitted to change this in the other charts. this is to prepare for renovatebot we will be adding in sometime soon to help with dependencies :)

Reply:
Done!